packages/theme/scripts/schema/generate.js

```diff
@@ -0,0 +1,69 @@
+import {jsonSchemaToZodDereffed} from 'json-schema-to-zod'
+import {get} from 'https'
+import {writeFile} from 'fs'
+import {resolve} from 'path'
+
+function remoteSchemaPath(schemaVersion) {
+  const repository = 'Shopify/theme-liquid-docs'
+  const schemaPath = `schemas/update/update_extension_schema_v${schemaVersion}.json`
+
+  return `https://raw.githubusercontent.com/${repository}/main/${schemaPath}`
+}
+
+function localSchemaPath(schemaVersion) {
+  return resolve(`./src/cli/services/update/schemas/update_extension_schema_v${1}.ts`)
```

Comment:
this might be just my gap in TS, but why pass `schemaVersion` and then use `1` to format a var in a string?

Reply:
Oops! Great catch! Thank you -- it's fixed now :)